doc/neps/nep-template.rst

```diff
@@ -24,6 +24,16 @@ the existing problem, who it affects, what it is trying to solve, and why.
 This section should explicitly address the scope of and key requirements for
 the proposed change.
 
+Usage and Impact
+----------------
+
+This section describes how users of NumPy will use features described in this
+NEP. It should be comprised mainly of code examples that wouldn't be possible
+without acceptance and implementation of this NEP, as well as the impact the
+proposed changes would have on the ecosystem. This section should be written
+from the perspective of the users of NumPy, and the benefits it will provide
+them; and as such, it should include implementation details only if
+necessary to explain the functionality.
```

Comment:
Perhaps worth re-iterating in this file, "This section will be sent out to the mailing list" or similar, wherever it fits best in this paragraph.

Reply:
Added, thanks.